### PR TITLE
DOCS: link to the mdformat-admon plugin

### PR DESCRIPTION
**Pull request by KyleKing**
I'm not sure if you're open to contributions here, but I thought adding a link to my new plugin might be useful!

### CODE REVIEW

File: docs/users/plugins.md [hunk 1]
```diff
@@ -128,4 +128,9 @@ formatted = mdformat.text(unformatted, extensions={"tables"})
     <td><code>deflist</code></td>
     <td>Adds support for <a href="https://pandoc.org/MANUAL.html#definition-lists">Pandoc-style</a> definition lists</td>
   </tr>
+  <tr>
+    <td><a href="https://github.com/KyleKing/mdformat-admon">mdformat-admon</a></td>
+    <td><code>admonitions</code></td>
```
Comment (KyleKing) [inline docs/users/plugins.md:133]: Should the rows in this table be sorted alphabetically or grouped?

Comment (hukkin) [inline docs/users/plugins.md:133]:
```suggestion
    <td><code>admonition</code></td>
```
I think this should match the string that is used here https://github.com/KyleKing/mdformat-admon/blob/2b55fb462fac5e4e6b1abfad1f3d27c75bd1209c/pyproject.toml#L36

Comment (hukkin) [reply docs/users/plugins.md:133]: I think this used to be in alphabetical order but no longer is, so I don't care much. The order can be fixed seprately for all items if seen necessary.

**Review by hukkin — CHANGES_REQUESTED** [review]
Thanks, this is very welcome addition to the docs!

File: docs/users/plugins.md (hunk 1 above)
Comment (KyleKing) [reply docs/users/plugins.md]: Thanks! I'm happy to make that change this evening, or you can go ahead and merge

**Pull request merged**